ckanext/googleanalytics/controller.py

```diff
@@ -146,45 +127,28 @@ def search(self, ver=None, register=None):
 
 class GAOrganizationController(OrganizationController):
     def _post_analytics(self, user, request_obj_type, request_function, request_id):
-        if toolkit.config.get('googleanalytics.id'):
-            data_dict = {
-                "v": 1,
-                "tid": toolkit.config.get('googleanalytics.id'),
-                "cid": hashlib.md5(user).hexdigest(),
-                # customer id should be obfuscated
-                "t": "event",
-                "dh": c.environ['HTTP_HOST'],
-                "dp": c.environ['PATH_INFO'],
-                "dr": c.environ.get('HTTP_REFERER', ''),
-                "ec": "CKAN Organization Page View",
-                "ea": request_obj_type + request_function,
-                "el": request_id,
-            }
-            plugin.GoogleAnalyticsPlugin.analytics_queue.put(data_dict)
-
-        if toolkit.config.get('googleanalytics.id2'):
```

Comment:
Looks like we removing necessary functionality here.

Reply:
We were not actively using the second Google Analytics tracking id so we're removing it.